go.sum

```diff
@@ -58,6 +58,8 @@ github.com/crossplaneio/crossplane v0.5.0-rc.0.20191211203849-05517d46525d h1:Xi
 github.com/crossplaneio/crossplane v0.5.0-rc.0.20191211203849-05517d46525d/go.mod h1:f/e99LDWClmhkvMuvK7UMP1g5b718DGmdnpdrflK0HI=
 github.com/crossplaneio/crossplane-runtime v0.2.4-0.20191211004842-fa83d075257c h1:Xmv6RSdmFf2WeBpzVaqEBVxmGVAAUxA61eMgrdQ+Vsw=
 github.com/crossplaneio/crossplane-runtime v0.2.4-0.20191211004842-fa83d075257c/go.mod h1:aVLQmXiPeb/jhGGQFIxx4Rc1CFC6xq2BA6A3neLZIHA=
+github.com/crossplaneio/crossplane-runtime v0.3.0 h1:d3937o6XhVUcm/bqgUPUjTo/OjVlAjh1c7rx4VPJoV8=
```

Comment:
any idea of why this crossplane-runtime  v0.3.0 entry is **added** to the `go.sum` instead of **replacing** the crossplane-runtime v0.2.4-xyz entries just above?

Reply:
all the other PRs replace the old version in `go.sum`, but this one doesn't 🤔 

e.g. https://github.com/crossplaneio/crossplane/pull/1135/files#diff-f949e2d81c8076ebbf8af38fcbb72c1f

Reply:
Should be fixed now!